jwst/regtest/test_fgs.py

```diff
@@ -0,0 +1,88 @@
+"""Regression tests for FGS Guidestar in ID and FINEGUIDE modes"""
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.lib.suffix import replace_suffix
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+def is_like_truth(rtdata, fitsdiff_default_kwargs, suffix, truth_path='truth/fgs/test_fgs'):
+    """Compare step outputs with truth"""
+    output = replace_suffix(
+        os.path.splitext(os.path.basename(rtdata.input))[0], suffix
+    ) + '.fits'
+    rtdata.output = output
+
+    rtdata.get_truth(os.path.join(truth_path, output))
+
+    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    assert diff.identical, diff.report()
+
+
+@pytest.fixture(scope='module')
+def run_fgs_imaging_pipelines(jail, rtdata_module):
+    """Run the pipelines"""
+    rtdata = rtdata_module
+    rtdata.get_data('fgs/level1b/exptype_fgs_image_uncal.fits')
+
+    collect_pipeline_cfgs('config')
+    args = [
+        'config/calwebb_detector1.cfg',
+        rtdata.input,
+        '--steps.group_scale.save_results=true',
+        '--steps.dq_init.save_results=true',
+        '--steps.saturation.save_results=true',
+        '--steps.superbias.save_results=true',
+        '--steps.refpix.save_results=true',
+        '--steps.firstframe.save_results=true',
+        '--steps.lastframe.save_results=true',
+        '--steps.linearity.save_results=true',
+        '--steps.dark_current.save_results=true',
+        '--steps.persistence.save_results=true',
+        '--steps.jump.save_results=true',
+        '--steps.ramp_fit.save_results=true',
+        '--steps.gain_scale.save_results=true',
+    ]
```

Comment:
Do you also need to set the --steps.jump.rejection_threshold param to a high value, so that it doesn't take 2.5 hours to run?

Reply:
Actually thought I had taken that out, since it did nothing about the ramp_fit runtime. Whatever is the reason for the long running there, it seems to be invariant against what jump found.

Reply:
Hmmm, so if it takes 2.5 hours to run, even when no jumps are flagged, then that's more incentive yet to not have the science-mode (FGS_IMAGE) test in there at all.

Reply:
Removing!